submit-queue/pod.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: submit-queue
+  namespace: default
+spec:
+  containers:
+  - image: gcr.io/google_containers/submit-queue:0.1
+    command:
+      - bash
+      - -c
+      - "/submit-queue --user-whitelist=/whitelist.txt --jenkins-host=http://jenkins-master:8079 --token=$(cat /etc/secret-volume/token) --alsologtostderr"
```

Comment:
Switched to 8080 (I swear it was 8080 before...)

wrt to the command line, that's true, and we should prob. fix eventually, but for now, if you can log onto the machine, you can talk to the kubernetes API server and read the secret anyway.